src/reconcile/rec_visibility.c

```diff
@@ -231,6 +231,8 @@ __get_next_rec_upd(WT_SESSION_IMPL *session, WT_UPDATE **inmem_upd_pos, bool *la
         if (*inmem_upd_pos != NULL)
             *inmem_upd_pos = (*inmem_upd_pos)->next;
     }
+    if (*updp != NULL && WT_TXNID_LT(*max_txn, (*updp)->txnid))
```

Comment:
Not 100% sure about this commit so you'll have to pay attention to this one. It seems to me that this is tracking the max transaction id of a page so we know if we can safely discard a page/tree. It seems to me that it shouldn't only apply to in-memory updates like it did before.